stacks/environment-local/lead.tf

```diff
@@ -48,13 +47,6 @@ module "toolchain" {
 
   harbor_registry_disk_size    = "200Gi"
   harbor_chartmuseum_disk_size = "100Gi"
-
-
-  smtp_host       = "mailhog"
-  smtp_port       = "1025"
-  smtp_username   = ""
-  smtp_password   = ""
-  smtp_from_email = "noreply@liatr.io"
```

Comment:
These should also be removed from the toolchain module's `variables.tf` if we aren't going to pass them anymore.